Lib/importlib/_bootstrap.py

```diff
@@ -826,10 +826,15 @@ def module_repr(m):
 
     @classmethod
     def find_spec(cls, fullname, path=None, target=None):
-        if _imp.is_frozen(fullname):
-            return spec_from_loader(fullname, cls, origin=cls._ORIGIN)
-        else:
+        info = _call_with_frames_removed(_imp.find_frozen, fullname)
+        if info is None:
             return None
+        data, ispkg = info
+        spec = spec_from_loader(fullname, cls,
+                                origin=cls._ORIGIN,
+                                is_package=ispkg)
+        spec.loader_state = data
```

Comment:
Not relevant for this PR/bug but `loader_state` is pretty poorly documented.  It's mentioned in `importlib.rst` but not even in `ModuleSpec`'s docstring.  It's only used in `LazyLoader` AFAICT.  We should probably better document this non-private attribute.

Reply:
https://bugs.python.org/issue45364 🙂